gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -186,7 +186,8 @@ def __init__(self,
                compression_type=CompressionTypes.AUTO,  # type: str
                buffer_size=DEFAULT_VCF_READ_BUFFER_SIZE,  # type: int
                validate=True,  # type: bool
-               allow_malformed_records=False  # type: bool
+               allow_malformed_records=False,  # type: bool
+               use_nucleus=False  # type: bool
```

Comment:
it's more future proof to make an enum for this like `VcfParserType` and use that here (in case we decide to add a third parser). There is even a 'simpler' way of just using the class string and let python actually construct it (so you dont even need an if statement) but that may be less readable.

Reply:
I think we should. It's very reasonable to have other types of parsers in the future (e.g. we're thinking about a bgzip parser; which may end up being a layer on top of Nuclues). In general, it's a good software engineering practice to not use bools for things that can possibly have more than two values.